lint.sh

```diff
@@ -77,6 +77,26 @@ check_bash(){
   done < <(find ${fargs[@]} -iname \*.sh)
 }
 
+check_naming_standards() {
+  # Limits lint to only gating unique files
+  # Excludes webhooktranslator as it has additional packaging
+  # and it's own tox
+  dirs_to_lint="pipeline_steps rpc_jobs scripts"
+  for _dir in ${dirs_to_lint}; do
+    for _file in $(find ${_dir} -type f -not -path '.*/.*'); do
+      python scripts/lint_naming_conventions.py \
```

Comment:
Passing the list of dirs to python does sound like a better idea. I started with approaching it in bash, but I thought the regex got a little too sloppy and would be easier to follow in python.